apps/jobs/views.py

```diff
@@ -2308,10 +2323,12 @@ def get_github_badge_data(
         if team_data["submission__participant_team"] == int(
             participant_team_pk
         ):
-            data["message"] = f"{challenge_obj.title} Rank #{idx+1}"
+            data["message"] = "{} Rank #{}".format(
+                challenge_obj.title, idx + 1
+            )
             break
         else:
-            data["message"] = f"{challenge_obj.title}"
+            data["message"] = "{}".format(challenge_obj.title)
```

Comment:
Maybe just leave the title as is? 
`data["message"] = "{}".format(challenge_obj.title)` -> `data["message"] = challenge_obj.title`

Reply:
@KhalidRmb This is a pre-commit change and unrelated with the pr. It was showing syntax error so I shifted to the format type. I think both the lines means same right?

Reply:
I understand and they do mean the same, but using string formatting here is redundant. Better to remove it

Reply:
@KhalidRmb Done the required change.